helm/kiam/CHANGELOG.md

```diff
@@ -1,4 +1,11 @@
 # Helm Chart Changelog
+
+## 3.1.0
+07 October 2019
+
+Notable Changes:
+* Update kiam release from 3.3 to 3.4
```

Comment:
Please add a link to this PR for the bullet point, e.g: [#304] - appreciate this can't be done until the PR has been opened.

Reply:
Fixed